app/src/main/scala/org/alephium/explorer/persistence/schema/CustomJdbcTypes.scala

```diff
@@ -97,18 +97,34 @@ object CustomJdbcTypes {
       bytes => ByteString.fromArrayUnsafe(bytes)
     )
 
-  @SuppressWarnings(Array("org.wartremover.warts.OptionPartial"))
   implicit val seqByteStringType: JdbcType[Seq[ByteString]] =
     MappedJdbcType.base[Seq[ByteString], Array[Byte]](
       byteStrings => serialize(AVector.unsafe(byteStrings.toArray)).toArray,
       bytes =>
-        deserialize[AVector[ByteString]](ByteString.fromArrayUnsafe(bytes)).toOption.get.toSeq
+        deserialize[AVector[ByteString]](ByteString.fromArrayUnsafe(bytes)) match {
+          case Left(error)  => throw error
+          case Right(value) => value.toSeq
+      }
```

Comment:
yes you are totally right, it's not the first time I shamefully wrote `toSeq`. 

In the explorer backend we deal with `Seq` (or [maybe IndexedSeq](https://github.com/alephium/explorer-backend/issues/148)) as slick is returning such collection, but when we are dealing with `alephium` model, we get the `AVector`. 
In the particular case here, I will have a look if we can deserialize a serialized `AVector` directly into a `Seq`, but it won't solve the `toSeq` inside the `BlockFlowClient` as we are receiving the models directly from `alephium`.
One solution to dig would be to use `AVector` everywhere, but I don't know if we can override slick to return an `AVector` instead of `Seq`, knowing slick my gut feeling is no, but I'll have a look, the goal is not to to do an `AVector.from(seq)` everywhere either....